tasks/template.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Filebeat template file(s)
+  copy:
+    src={{ item }}
```

Comment:
Instead of using = here, you can make this more lintable with the field: value syntax.